File: Programming_Basics_with_Python/while_loop_exercise/01._old_books.py
searched_book = input()
books_counter = 0
book_is_found = False
next_book = input()
while next_book != "No More Books":
    if next_book == searched_book:
        book_is_found = True
        break
    books_counter += 1
    next_book = input()
if book_is_found:
    print(f"You checked {books_counter} books and found it.")
else:
    print("The book you search is not here!")
    print(f"You checked {books_counter} books.")

File: Programing_Basics_with_JS/whileLoopExercise/01.OldBooks.js
function oldBooks(input) {
    let index = 0;
    let searchedBook = input[index];
    index++;
    let command = input[index];
    index++;

    let checkedBook = 0;
    let bookIsFound = false;

    while(command !== 'No More Books') {
        checkedBook++;
        let findBook = command;

        if (findBook === searchedBook) {
            bookIsFound = true;
            checkedBook--;
            break;
        }

        command = input[index];
        index++;
    }

    if(bookIsFound) {
        console.log(`You checked ${checkedBook} books and found it.`);
    } else {
        console.log('The book you search is not here!');
        console.log(`You checked ${checkedBook} books.`);
    }
}

oldBooks(["Troy",
"Stronger",
"Life Style",
"Troy"])

oldBooks(["The Spot",
"Hunger Games",
"Harry Potter",
"Torronto",
"Spotify",
"No More Books"])

oldBooks(["Bourne",
"True Story",
"Forever",
"More Space",
"The Girl",
"Spaceship",
"Strongest",
"Profit",
"Tripple",
"Stella",
"The Matrix",
"Bourne"])


